scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java

```diff
@@ -97,14 +105,18 @@ public Scheduler login(Credentials cred) throws LoginException, AlreadyConnected
         String user = unPrincipal.getName();
 
         logger.info("user : " + user);
+
+        // get AWS S3 credentials
+        getAWSCredentials();
```

Comment:
// get AWS S3 credentials
getAWSCredentials(); 

You could rename the method (if it only gets S3 creds)  into e.g. getAwsS3Credentials(); Then you could save the comment on top of it.